ts/BUILD.bazel

```diff
@@ -1,10 +1,33 @@
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@bazel_skylib//rules:common_settings.bzl", "int_flag")
 
 exports_files(
     glob(["*.bzl"]),
     visibility = ["//docs:__pkg__"],
 )
 
+# Enable with --@aspect_rules_ts//ts:skip_lib_check=1
+int_flag(
+    name = "skip_lib_check",
+    # Default is 0 to be a non-breaking change in rules_ts 1.x
+    # TODO(2.0): change default to -1 so we require users to consider which value of the flag is best.
+    build_setting_default = 0,
```

Comment:
`-1` should be the default. 1/0 True/False  -1/honor ts_config

Reply:
PTAL with a pair of bool_flags instead, even less confusing than any int scheme